src/lfc_scripts.py

```diff
@@ -1,13 +1,61 @@
 import mwclient
 import config
+from geopy.geocoders import Nominatim
+import folium
 
-site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
-site.login(config.username, config.api_key)
+def main():
+    site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
+    site.login(config.username, config.api_key)
 
-response = site.api(
-    'torquedataconnect',
-    format='json',
-    path='/competitions/LoneStar2020/proposals'
-)
+    response = site.api(
+        'torquedataconnect',
+        format='json',
+        path='/competitions/LoneStar2020/proposals'
+    )
+    proposal_ids = response["result"]
 
-print(response["result"]) 
\ No newline at end of file
+    geolocator = Nominatim(user_agent = "map")
+
+    #Extract organization locations
```

Comment:
One handy rule of thumb I learned is that if you're writing a comment to explain a block of code, it probably makes sense to put that block in a named method.

You want your functions to be as short as possible -- this one is a little bit long and it's because you have sections like this which could be encapsulated in their own method.

I know you're planning to make these kinds of revisions / engineer what you've done a bit more, so no need to change as part of this PR.  Over time you should try to actually define these kinds of methods as part of the original PR however -- your WIP implementation doesn't have to be factored, but by the time it gets merged to the main branch it should reflect that kind of organization.

So in this case you might have a few methods like:

* `extract_locations` which is written a bit more generally to in a result set and a resolver function which returns the location that you want extracted (in this case concat_org_location would be the resolver)
* `get_latlong` which takes in a location string and returns a latlng

And later you might have a:

* `generate_map_plot` or `generate_map` which takes an array of latlngs

Reply:
Noted, will be sure to change this in future iterations.